client/tx/search.go

```diff
@@ -53,12 +57,19 @@ $ gaiacli query txs --tags '<tag1>:<value1>&<tag2>:<value2>'
 				}
 
 				keyValue := strings.Split(tag, ":")
-				tag = fmt.Sprintf("%s='%s'", keyValue[0], keyValue[1])
+				if keyValue[0] == types.TxHeightKey {
+					tag = fmt.Sprintf("%s=%s", keyValue[0], keyValue[1])
```

Comment:
does this panic if you set it as `%s='%s'` ?

Reply:
yes, this piece of tendermint code would panic on string type cannot be converted into int64...

tendermint/state/txindex/kv/kv.go

```
// lookForHeight returns a height if there is an "height=X" condition.
func lookForHeight(conditions []query.Condition) (height int64) {
	for _, c := range conditions {
		if c.Tag == types.TxHeightKey && c.Op == query.OpEqual {
			return c.Operand.(int64)
		}
	}
	return 0
}
```